easybuild/easyblocks/o/openblas.py

```diff
@@ -13,8 +14,31 @@ class EB_OpenBLAS(ConfigureMake):
     def build_step(self):
         """Custom build procedure for OpenBLAS."""
 
+        self.cfg['buildopts'] += 'BINARY=64 USE_THREAD=1 USE_OPENMP=1 CC="%s" FC="%s"' % (
+            os.environ['CC'], os.environ['FC'])
```

Comment:
@edmondac Ideally, this is done a bit more carefully. If the easyconfig already specifies `BINARY=...`, then we shouldn't hard inject `BINARY=64` here. Likewise for the other stuff.

Also, you must use `self.cfg.update('buildopts', "...")` here, `+=` doesn't do what you think it does (the stuff you're injecting will not be picked up because of underlying implementation details related to templating).